go/client/stellar_common.go

```diff
@@ -41,11 +41,15 @@ func printPayment(g *libkb.GlobalContext, p stellar1.PaymentCLILocal, verbose bo
 	}
 	var to string
 	switch {
+	case p.ToUsername != nil && p.ToAssertion != nil:
```

Comment:
Maybe you want to add `&& *p.ToUsername != *p.ToAssertion`? Because isn't it the case that in the usual path the sender posts the ToAssertion as the username they're sending to?

Reply:
hmm in the case where recipient has no wallet? good idea, I missed this

Reply:
I meant regardless of whether the recipient has a wallet, I thought that assertion=username was posted. Why would whether the recipient had a wallet affect it?

Reply:
my understanding of when ToAssertion is set (or returned) is a bit messy then. I thought it's only for relay payments hence if the recipient has a wallet, it's never a relay, so this field will never be used there

Reply:
Oh yeah you're absolutely right. Yes, for that case then :D